unstructured/ingest/interfaces.py

```diff
@@ -201,6 +202,13 @@ def version(self) -> t.Optional[str]:
             self.update_source_metadata()
         return self.source_metadata.version  # type: ignore
 
+    @property
+    def permissions_data(self) -> t.Optional[t.List[t.Dict[str, t.Any]]]:
+        """Role based access control data, aka permissions or sharing, from the source system."""
```

Comment:
```suggestion
        """Access control data, aka permissions or sharing, from the source system."""
```
probably no reason to assume this permission data is role based